src/practice.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import App from './app.js'
+
+export default class Practice extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      showAnswer: false,
+      currentIndex: 0
+    }
+    this.onShowClick = this.onShowClick.bind(this)
+    this.onNextClick = this.onNextClick.bind(this)
+  }
+
+  onShowClick() {
+    this.setState(prevState => ({
+      showAnswer: !prevState.showAnswer
+    }))
+  }
+
+  onNextClick(event) {
+    if (event.target.id === 'left') {
```

Comment:
Try to use `data-` attributes instead of `id` attributes for JavaScript metadata.